services/api/src/game/game-room.service.ts

```diff
@@ -2,13 +2,14 @@ import WebSocket, { WebSocketServer } from 'ws';
 import { v4 as uuidv4 } from 'uuid';
 import logger from '../logger';
 import { Player } from '../domain';
-import { PlaceFigureMessage } from '../domain/messages';
+import { MoveSkippedMessage, PlaceFigureMessage } from '../domain/messages';
 
 interface GameRoom {
   roomId: string;
   server: WebSocketServer;
   players: Map<WebSocket, Player>;
   turns: PlaceFigureMessage[];
```

Comment:
Don't we have a separate type for `Turn` that would indicate that it is actually a Turn, not a `FigureMessage`?